go.mod

```diff
@@ -151,4 +144,18 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+require (
```

Comment:
is this new block here intentional? do we want to split our dependencies into further groups?

Reply:
I think that was my IDE. I did not change anything.